src/server/models/Contest.ts

```
@@ -0,0 +1,53 @@
+	endDateAndTime: Date,
+	timestamp: Date,
+	rules: {
+		pointsLostPer: {
+			interval: Number,
+			intervalAmount: Number,
+			wrongAnswers: Number,
+			wrongAnswersAmount: Number,
+		},
+		minimumPointsPerProblem: Number,
+	},
+	participants: Array<String>,
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)